src/Microsoft.AspNetCore.Dispatcher/Patterns/InlineRouteParameterParser.cs

```diff
@@ -3,12 +3,13 @@
 
 using System;
 using System.Collections.Generic;
+using System.Linq;
 
-namespace Microsoft.AspNetCore.Dispatcher
+namespace Microsoft.AspNetCore.Dispatcher.Patterns
 {
     public static class InlineRouteParameterParser
```

Comment:
This class is totally broken. Everything here should be a detail of the parser. I logged #475 to follow up on this